src/pages/organizations/[organizationid].js

```diff
@@ -158,9 +158,13 @@ export default function Organization(props) {
             <Box sx={{ mt: 2 }}>
               <Info
                 iconURI={CalendarIcon}
-                info={`Organization since ${moment(
-                  organization?.created_at,
-                ).format('MMMM DD, YYYY')}`}
+                info={
+                  <time
+                    dateTime={organization?.created_at}
+                  >{`Organization since ${moment(
```

Comment:
Final improvement, the text **Organization since**  is inside the `time` element this should not be the case. I'd recommend a fragment.